shell/platform/linux/fl_view_accessible.cc

```diff
@@ -9,14 +9,33 @@
 struct _FlViewAccessible {
   GtkContainerAccessible parent_instance;
 
+  FlEngine* engine;
+
   // Semantics nodes keyed by ID
   GHashTable* semantics_nodes_by_id;
 };
 
+enum { PROP_0, PROP_ENGINE, PROP_LAST };
+
 G_DEFINE_TYPE(FlViewAccessible,
               fl_view_accessible,
               GTK_TYPE_CONTAINER_ACCESSIBLE)
 
+static void fl_view_accessible_set_engine(FlViewAccessible* self,
+                                          FlEngine* engine) {
+  self->engine = engine;
```

Comment:
Probably a good idea to put `assert(self->engine == null)` before this to catch any future bugs that might be introduced.

Reply:
Done + static functions renamed like in https://github.com/flutter/engine/pull/33100#discussion_r865392252